### DISCUSSION TITLE
Disabling installation of Administrator Launcher during unattended installation

### DISCUSSION THREAD

**Discussion by wtellis** [nextgenhealthcare/connect#6061] [Q&A]
We're looking to automate updating our Mirth installations but have run into an issue with using the Windows installer.  We've tried running it in unattended mode via the "-q" switch but it keeps launching the wizard to install the Mirth Administrator Launcher which then stops the process since it needs user interaction. We've tried passing in a response file that we copied from one our our existing Mirth installations, but its not clear which parameter disables this step. Is there a way to disable the installation of the Admin Launcher when running in unattended mode?

**Comment by jonbartels** [comment]
Quick disclaimer - I am a Mac user, I almost never use the installer, and the Install4J config for Mirth is not open sourced. 

I think you might be able to combine two things to get what you need:

1. https://www.ej-technologies.com/resources/install4j/help/doc/installers/options.html - Try running the installer in `console` mode. Does that prompt you to run/not run the MCAL installer?
2. According to this link https://www.ej-technologies.com/resources/install4j/help/doc/installers/responseFile.html:
> A response file is generated automatically after an installation is finished. The generated response file is found in the .install4j directory inside the installation directory and is named response.varfile. When you request debugging information from a user, you should request this file in addition to the installer log file.

That varfile can then be imported with the `-varfile` option in a subsequent installation to use the same options. I think.

I *think* this gets you what you want to do unattended installations, the same way, each time.
